input/pagecontent/index.md

```diff
@@ -10,9 +10,9 @@ The focus AU eRequesting Release 1 (R1) is support of pathology and medical imag
 
 This implementation guide is under development through the AU eRequesting project as part of the [Sparked AU FHIR Accelerator](https://sparked.csiro.au). The Sparked AU FHIR Accelerator is a community comprising government, technology vendors, provider organisations, peak bodies, practitioners, and domain experts, to accelerate the creation and use of national FHIR standards in health care information exchange.
 
-The AU eRequesting project includes:
-- AU eRequesting Data for Interoperability
-- AU eRequesting FHIR Implementation Guide
+The Sparked AU FHIR Accelerator includes:
+- AU eRequesting Data for Interoperability - clinician focussed data requirements
+- AU eRequesting FHIR Implementation Guide - HL7 Australia technical data access and exchange specification
 
 The [Australian eRequesting Data for Interoperability (AUeReqDI)](https://sparked.csiro.au/index.php/sparked-products-resources/auereqdi/auereqdi-release-1/) is focused on an agreement of the minimum data required to support standardised eRequesting within the Australian health context, and forms a common language foundation that allows systems to exchange semantically accurate data for eRequests. AUeReqDI outputs form a set of data requirements to be considered and referred to as part of the development and definition of AU eRequesting. 
 
```

Comment:
Commenting here to get an inline comment. 
Please update AU Core Placer description in 1.4 AU eRequesting Actors as well.